test/code_sandbox.test.js

```diff
@@ -1,32 +1,32 @@
 const sandbox = require('../modules/code_sandbox_manager.js');
 const assert = require('assert');
 
-describe('Code Sandbox', function () {
-  describe('alert()', function () {
-    it(`should be connected to sandbox host's stdout`, function () {
+describe('Code Sandbox', async() => {
+  describe('alert()', async() => {
+    it(`should be connected to sandbox host's stdout`, async() => {
       const code      = `alert('Hello there!');`;
-      const result    = sandbox.run(code);
+      const result    = await sandbox.run(code);
 
       assert.equal(result.stdout, 'Hello there!\n');
     });
   });
 
-  describe('alert()', function () {
-    it(`should be connected to sandbox host's stderr`, function () {
+  describe('alert()', async() => {
```

Comment:
nit: is there a reason this isn't combined with the first `describe('alert()')` block?

Reply:
Whoops, thanks for the catch!
Looks like it got checked-in on a previous PR where I was still learning `mocha` and misunderstood how `describe()` works. Fixed now :)